lib/helpers.js

```diff
@@ -107,6 +107,27 @@ export async function getAssetSizes() {
   return prAssets;
 }
 
+export function sumAssetSizes(assetSizeReport) {
+  return Object.entries(assetSizeReport).reduce(
+    (acc, [filename, fileMeta]) => {
+      let fileType;
+      if (filename.endsWith('.js')) {
+        fileType = 'js';
+      } else if (filename.endsWith('.css')) {
+        fileType = 'css';
+      } else {
+        return acc;
+      }
```

Comment:
I've currently got this hardcoded to support `js` and `css`.

If we wanted more file types, or for file types to be handled more dynamically, I can amend.

I've only witnessed this action reporting JS and CSS, so I stuck with simple `if` statements for now.